invenio_rdm_records/records/jsonschemas/records/parent-v2.0.0.json

```diff
@@ -67,6 +73,38 @@
               }
             }
           }
+        },
+
+        "settings": {
+          "description": "Access settings for the record.",
+          "type": "object",
+          "additionalProperties": false,
+          "properties": {
+            "user_requests_enabled": {
+              "description": "Whether or not access requests are enabled for authenticated users.",
+              "type": "boolean"
+            },
+            "guest_requests_enabled": {
+              "description": "Whether or not access requests are enabled for unauthenticated guests.",
+              "type": "boolean"
+            },
+            "access_conditions_text": {
+              "description": "Custom description for access conditions to be displayed rather than the default text.",
+              "type": ["string", "null"]
+            },
+            "default_access_permission": {
+              "description": "The default permission to give out via access requests.",
+              "type": "string"
+            },
+            "default_link_expiration": {
+              "description": "The default time period before secret links expire.",
+              "type": ["string", "null"]
+            },
+            "default_request_receiver": {
+              "description": "The default receiver for access requests.",
+              "type": ["string", "null"]
```

Comment:
Per RFC, we want to have the possibilities to set **"record owner"** or **"managers/owners of community X"** (as long as the record is part of that community).

This could be encoded as a sort of **reference dictionary** (e.g. `{"user": "1337"}`, or `{"community": "XCOM"}`).
Of course, then we would need to ensure that this can't be set to just about any arbitrary user or community while setting the value (and we would need to re-validate whenever the record ownership or the record's communities change).

Alternatively, we could use a sort of simple string here (like `"record_owner"` or `"community:XCOM"`), but that's probably just strictly worse than the reference dictionary.

Ideas, thoughts, prayers?

Reply:
Uhh... I interpreted the [RFC](https://codimd.web.cern.ch/U6_FMlwPS9ShGVtYDBjIUQ?view) in a way that you could select **a** community as alternative receiver, whose manager members would receive the requests.
Re-reading it, I still _think_ that's the intended semantics but I'm not 100% sure anymore.
Ping @lnielsen